submissions/cynthiaNwakaeme/CSS/product.css

```diff
@@ -11,84 +11,75 @@ body{
     margin: auto;
     overflow: hidden;
 }
-ul{ 
+ul, h2, h3, p{ 
     padding: 0;
     margin: 0;
 }
+li a{
```

Comment:
@ToluAkin  here is where i declared it. For some reasons it is not showing in that session you commented on.